app/assets/stylesheets/application.scss

```diff
@@ -5,10 +5,15 @@
 @import "base";
 @import "components/*";
 
+@import "static";
+
 article {
   @extend .container-fluid;
 }
 
 section {
   @extend .row;
+
+  margin-left: 0;
+  margin-right: 0;
```

Comment:
As long as this is in a container it should be good. Just one of those bootstrap issues.